pack/tdc/tdcc.json

```diff
@@ -1 +1,425 @@
-[]
+[
+    {
+        "back_link": "11253b",
```

Comment:
I think we usually add 500 to all of the investigator cards, so they stay in the cycle.

So this should be 11753b